components/src/molecules/product-list-item/ProductListItem.stories.js

```diff
@@ -0,0 +1,96 @@
+import { storiesOf } from '@storybook/vue'
+import { action } from '@storybook/addon-actions'
+
+import ALabel from '../../atoms/label/Label.vue'
+import AProductListItem from './ProductListItem.vue'
+import AQuantityUpdate from '../../molecules/quantity-update/QuantityUpdate.vue'
+
+const info = 'Product List Item component'
+
+// @vue/component
+const defaultData = {
+  components: {
+    ALabel,
+    AProductListItem,
+    AQuantityUpdate
+  },
+  data () {
+    return {
+      qty: 1,
+      options: [
+        {
+          'id': 'size_id_1',
+          'label': 'Size',
+          'value': 'XXL'
+        },
+        {
+          'id': 'color_id_1',
+          'label': 'Color',
+          'value': 'Red'
+        }
+      ]
+    }
+  }
+}
+
+storiesOf('Molecules/Product List Item', module)
+  .addParameters({ info })
+  .add('Default', () => ({
+    ...defaultData,
+    methods: {
+      removeMethod: action('Remove')
+    },
+    template: `
+      <a-product-list-item
+        name="Ingrid Running Jacket"
+        imageUrl="/images/product/cart-product-160_160.jpg"
+        price=""
+        specialPrice="$100,00"
+        oldPrice="$299,99"
+        url="#"
+        :options="options"
+        @remove="removeMethod"
+      />
+    `
+  }))
+  .add('With Qty', () => ({
```

Comment:
Imo we dont need this story. Instead of that we should have story 'With slots', where we use all slots (also with quantity-update). 
The second point is: maybe we should add default quantity-update inside the component and then overwrite it with named slots ?

Reply:
Ok, I replace it with `Without qty` scenario, also qty named slot was updated. Please check and let me know what do you think. thanks